run-dev.sh

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 . venv/bin/activate
-parcel watch app/js-src/index.ts --dist-dir app/static/js &
+# parcel watch app/js-src/index.ts --dist-dir app/static/js &
```

Comment:
Is this meant to be disabled? If so, why? I thought we needed this when developing ts.

Reply:
I'm trying to move away from a build stage on frontend since its not a javascript app and it's piecemeal. One less dependency for development.

Reply:
I think the benefit of simple build environment overrides ability to write typescript. also there is prolly a shim layer that converts typescript to javascript at runtime if we really wanna write it. I'm actually not against just vanilla javascript.